### PR TITLE
Exporter Landing page

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: web/web.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package web
+
+import (
+	"image/color"
+	"net/http"
+)
+
+// Config represents the configuration of the web listener.
+type Config struct {
+	Color       color.Color // Used for the landing page header
```
Comment (brian-brazil) [inline web/web.go:23]: Does this need to be configurable?

File: web/web.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	Name        string      // The name of the exporter, generally suffixed by _exporter
```
Comment (brian-brazil) [inline web/web.go:24]: Full stops

File: web/web.go (hunk 1 above)
Comment (roidelapluie) [reply web/web.go:23]: Thanks for the very early review. I think that having just a different color for each exporter would be a nice 'touch'; without going to the point where we offer themes.

Comment (SuperQ) [reply web/web.go:23]: Yes, it's nice idea. For example, using the MySQL brand colors for the mysqld_exporter. :smile:

**Comment by SuperQ** [comment]
Ping, want to get this basic version checked in? I would like to add to it.

**Comment by roidelapluie** [comment]
Hello, at this stage we still need CSS. You should have pisg access to the branch if you want to help I think.

File: web/generated_assets.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            {{ range .Links }}
+            <li><a href="{{ .Address }}">{{.Text}}</a>{{ if .Description }}: {{.Description}}{{end}}</li>
+            {{ end }}
+            </ul>
+        </nav>
+    </body>
+</html>
+`
+var cssContent = `Hello
+`
```
Comment (SuperQ) [inline web/generated_assets.go:37]: How about something simple to start.

```suggestion
var cssContent = `
h1 {
   color: #fff;
   background-color: #343a40
}
`
```

**Comment by SuperQ** [comment]
Ping, it'd be nice to get the basics in so we can continue to build on it.

**Comment by SuperQ** [comment]
Closing in favor of https://github.com/prometheus/exporter-toolkit/pull/51

**Pull request closed without merging**